### ISSUE TITLE
 Only "chest" type can have a custom size, "furnace" always have a size of 2.

### ISSUE THREAD

**Issue by guikaua12** [DevNatan/inventory-framework#278]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### 🐛 Describe the bug

When opening the view to a player, this is throw:
` Only "chest" type can have a custom size, "furnace" always have a size of 2. Remove the parameter that specifies the size of the container on me.approximations.quantumWorkbench.view.ElectricFurnaceView or just set the type explicitly.`

View class:
```java
public class ElectricFurnaceView extends View {
  public ElectricFurnaceView() {
    super(ViewType.FURNACE);
  }
}
```

### ✔️ Expected behavior

Open a furnace GUI

### 👣 Steps to Reproduce

Register a furnace ViewType view and open it to the player.

### 💻 Platform

- [X] Bukkit
- [ ] Sponge
- [ ] Other

### ⭐ Versions

MC 1.18.2 IF 6f6a634cc5

**Comment by DevNatan** [comment]
Fixed in v3.0.0-beta (unreleased)
```java
public class ElectricFurnaceView extends View {

    @Override
    public void onInit(ViewConfigBuilder config) {
        config.type(ViewType.FURNACE);
    }
}
```

**Comment by DevNatan** [comment]
Fixed in v3.0.0-EAP.
